baybe/surrogates/base.py

```diff
@@ -83,6 +86,34 @@ def to_botorch(self) -> Model:
 
         return AdapterModel(self)
 
+    @staticmethod
+    def _get_parameter_scaler(
+        parameter: Parameter,
+    ) -> ScalerProtocol | Literal["passthrough"]:  # noqa: F821
```

Comment:
What is the error code that you are `noqa`-ing here?

Reply:
Please ignore, has become obsolete and is now removed from the old commits